docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -231,6 +231,65 @@ image::images/obs-ai-logs.gif[Observability AI assistant example, 75%]
+
+You can use the Observability AI Assistant connector to add AI-generated insights and custom actions to your alerting workflows.
+To do this:
+
+. <<create-alerts-rules,Create (or edit) an alerting rule>> and specify the conditions that must be met for the alert to fire.
+. Under **Actions**, select the **Observability AI Assistant** connector type.
+. In the **Connector** list, select the AI connector you created when you set up the assistant.
+. In the **Message** field, specify the message to send to the assistant:
++
+[role="screenshot"]
+image::images/obs-ai-assistant-action-high-cpu.png[Add an Observability AI assistant action while creating a rule in the Observability UI]
+
+You can ask the assistant to generate a report of the alert that fired,
+recall any information or potential resolutions of past occurrences stored in the knowledge base,
+and also include other active alerts that may be related.
+As a last step, you can ask the assistant to trigger an action,
+such as sending the report (or any other message) to a Slack webhook.
+
+NOTE: Currently you can only send messages to a Slack webhook.
+Support for additional actions will be added in the future.
+
+When the alert fires, contextual details about the event—such as when the alert fired,
+the service or host impacted, and the threshold breached—are sent to the AI Assistant,
+along with the message provided during configuration.
+The AI Assistant runs the tasks requested in the message and creates a conversation you can use to chat with the assistant:
+
+[role="screenshot"]
+image::images/obs-ai-assistant-output.png[AI Assistant conversation created in response to an alert]
+
+IMPORTANT: Conversations created by the AI Assistant are public and accessible to every user with permissions to use the assistant.
+
+It might take a minute or two for the AI Assistant to process the message and create the conversation.
+
+Note that overly broad prompts may result in the request exceeding token limits.
+For more information, refer to <<obs-ai-token-limits>>.
+Also, attempting to analyze several alerts in a single connector execution may cause you to exceed the function call limit.
+If this happens, modify the message specified in the connector configuration to avoid exceeding limits.
+
+When asked to send a message to another connector, such as Slack,
+the AI Assistant attempts to include a link to the generated conversation. For example:
+
+[role="screenshot"]
+image::images/obs-ai-assistant-slack-message.png[Message sent by Slack by the AI Assistant includes a link to the conversation]
+
+TIP: The `server.publicBaseUrl` setting must be correctly specified under {kib} settings,
+or the AI Assistant is unable to generate this link.
+
+The Observability AI Assistant connector is called when the alert fires and when it recovers.
+
+To learn more about alerting, actions, and connectors, refer to <<create-alerts>>.
```

Comment:
Not sure why, but this link is showing in italics for me.

![image](https://github.com/elastic/observability-docs/assets/114418652/fd7f0445-9165-4711-9fde-87b9c378176a)

Reply:
Yeah. The joys of asciidoc. If you don't include link text, the link is formatted in italics. If you do include link text, like `<<create-alerts,Alerting>>`, then the link is formatted normally.